### PR TITLE
Upgrade all Flask dependencies

### PR DESCRIPTION
**Pull request by sarayourfriend**
## Description of Changes

Upgrades all flask dependencies and removes some transient dependencies.

## Tests and linting

 - [ ] I have rebased my changes on current `develop`

 - [ ] pytests pass in the development environment on my local machine

 - [ ] `flake8` checks pass

### CODE REVIEW

File: OpenOversight/app/__init__.py [hunk 1]
```diff
@@ -110,6 +110,10 @@ def get_age_from_birth_year(birth_year):
     app.cli.add_command(advanced_csv_import)
     app.cli.add_command(use_original_image_for_faces)
 
+    @limiter.request_filter
+    def _endpoint_whitelist():
+        return request.endpoint == "static"
+
```
Comment (sarayourfriend) [inline OpenOversight/app/__init__.py:116]: https://github.com/alisaifee/flask-limiter/issues/285

File: requirements.txt [hunk 2]
```diff
@@ -1,22 +1,21 @@
-Flask==1.0.2
+Flask==2.0.1
 python-dotenv==0.7.1
 boto3==1.17.101
-werkzeug==1.0.1
```
Comment (sarayourfriend) [inline requirements.txt:4]: transient

File: requirements.txt [hunk 3]
```diff
@@ -1,22 +1,21 @@
...
+Flask-WTF==0.15.1
 Flask-Sitemap==0.3.0
 Pillow==7.2.0
 psycopg2-binary==2.9.1
 python-dateutil==2.8.1
-SQLAlchemy==1.3.0
+SQLAlchemy==1.4.21
 gunicorn==19.9
 Fabric3==1.14.post1
-itsdangerous==0.24
```
Comment (sarayourfriend) [inline requirements.txt:19]: transient

**Review by AetherUnbound — APPROVED** [review]
Assuming the app runs okay, this looks good!

**Pull request merged**